varlink/error.py

```diff
@@ -64,9 +71,8 @@ class InterfaceNotFound(VarlinkError):
 
     @classmethod
     def new(cls, message, namespaced=False):
-        return cls(
-            namespaced and message["parameters"].interface or message["parameters"].get("interface", None)
```

Comment:
Having looked at the `namespaced` stuff, I think that might be more readably put as
```python
message["parameters"].interface if namedspaced else message["parameters"].get("interface")
```
also dropping the `None` default for `.get`, since it's, well, the default. (Or the equivalent on the right hand side)